vendors/marvell/boards/mw300_rd/aws_demos/config_files/iot_config.h

```diff
@@ -37,20 +37,21 @@
  * level for all libraries; the library-specific settings override the global
  * setting. If both the library-specific and global settings are undefined,
  * no logs will be printed. */
-#define IOT_LOG_LEVEL_GLOBAL                    IOT_LOG_INFO
```

Comment:
Do we need these log level changes ?

Reply:
Yes. Reducing the log levels also helped in reducing the memory need in Heap. The logging task mallocs buffers for handling logs.